packages/actor-query-parse-sparql/lib/ActorQueryParseSparql.ts

```diff
@@ -4,8 +4,10 @@ import { KeysInitQuery } from '@comunica/context-entries';
 import type { IActorTest, TestResult } from '@comunica/core';
 import { failTest, passTestVoid } from '@comunica/core';
 import type { ComunicaDataFactory } from '@comunica/types';
+import type * as RDF from '@rdfjs/types';
+import { Sparql12Parser as SparqlParser } from '@traqula/engine-sparql-1-2';
```

Comment:
For easier pluggability, I wonder if it wouldn't be easier if all `@traqula/engine-` packages just expose a `Parser` class (so it's always the same name).
This is also the approach we follow with comunica engines.

Reply:
(this comment is not urgent btw)